aubase410/initializer.bal

```diff
@@ -15,12 +15,12 @@
 // under the License.
 
 // AUTO-GENERATED FILE.
-// This file is auto-generated by Ballerina.
+// This file is auto-generated by WSO2 Healthcare.
```

Comment:
This should be ballerina.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/6b7d2c631f1a7afe2fb84f099cdaee5897249841